python/python/tests/test_fts.py

```diff
@@ -79,6 +81,39 @@ def test_search_index(tmp_path, table):
     assert len(results[1]) == 10  # _distance
 
 
+def test_search_ordering_field_index_table(tmp_path, table):
+    table.create_fts_index("text", ordering_field_names="count")
+    rows = (
+        table.search("puppy", ordering_field_name="count")
+        .limit(20)
```

Comment:
get a big number, chances of randomly sampling 12-20 rows in order are basically 0.